src/app/utils/field-validation/answerField.factory.ts

```diff
@@ -0,0 +1,24 @@
+import { BasicField, IColumnSchema } from '../../../types'
+
+type ColumnWithFieldProperties = IColumnSchema & {
+  getQuestion: { (): string }
+  description: string
+  disabled: boolean
+  fieldType: BasicField.ShortText | BasicField.Dropdown
+}
+
+/**
+ * Takes a table field column and generates a form field by
+ * filling in the missing attributes.
+ * */
+export const createAnswerFieldFromColumn = (
+  column: IColumnSchema,
+): ColumnWithFieldProperties => {
+  const columnField = column
```

Comment:
this isn't an object copy, the original `column` variable is still being mutated. You could do this:

```typescript
return Object.assign({}, column, { disabled: false, ... })
```

See https://www.javascripttutorial.net/object/3-ways-to-copy-objects-in-javascript/

Reply:
made changes to use spread syntax